port/win/win_jemalloc.cc

```diff
@@ -1,7 +1,7 @@
 //  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
-//  This source code is licensed under the BSD-style license found in the
-//  LICENSE file in the root directory of this source tree. An additional grant
-//  of patent rights can be found in the PATENTS file in the same directory.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
```

Comment:
Was not that file under a BSD from LevelDB originally?

Reply:
I am not sure if this file came from LevelDB. Even if it did, we still have the BSD-related text on lines 6-8, similar to all LevelDB-imported files.

Reply:
I believe these are mostly from relatively new files, and from open-source community. They probably tested with older versions in their branches, and forgot to update the files. This specific file was from a pull request of yuslepukhin. All his other files followed the new header. So I believe it is mistake.